test/unit_tests/test_ProxyVerifier.cc

```diff
@@ -70,6 +70,53 @@ TEST_CASE("RuleChecks of non-duplicate fields", "[RuleCheck]")
     CHECK(equal_check_blank->test(key, test_name, empty_value));
     CHECK_FALSE(equal_check_blank->test(key, test_name, non_empty_value));
   }
+
+  SECTION("contains checks")
+  {
+    swoc::TextView contained_value("V");
```

Comment:
Maybe make this "Val" just to verify this works with more than just a character for some reason.